### DISCUSSION TITLE
Initialize MOM6 with alternate file, including UV

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1708] [Q&A]
Hello,

I am interested in initializing MOM6 in UFS using an initial condition file our group has generated from ORAS5, which includes U and V as well as TS. In the documentation [here](https://ufs-weather-model.readthedocs.io/en/latest/InputsOutputs.html) it only lists TS. Is it also possible to initialize the currents outside of using a restart file? 

Thanks,
Ben

**Comment by pjpegion** [comment]
Yes, Here is the block that you need to have in MOM_input:

! === module MOM_state_initialization ===
INIT_LAYERS_FROM_Z_FILE = False  !   [Boolean] default = False
                                ! If true, initialize the layer thicknesses, temperatures, and salinities from a
                                ! Z-space file on a latitude-longitude grid.
THICKNESS_CONFIG        = "thickness_file"
THICKNESS_FILE          = "ORAS5.mx025.ic.nc"
TS_CONFIG               = "file"
TS_FILE                 = "ORAS5.mx025.ic.nc"
TEMP_IC_VAR             = "Temp"
VELOCITY_CONFIG         = "file"
VELOCITY_FILE           = "ORAS5.mx025.ic.nc"

and set
input_filename = n  in the &MOM_input_nml part of input.nml
